### PR TITLE
{CI} Optimize pipeline result

### PR DESCRIPTION
**Pull request by wangzelin007**
**Related command**

**Description**

Now we will directly display the test results of all modules, the comment is too long.
So I add a new level `TestName` to optimize the display effect of azure-cli full test: 
image
TODO: 
Support only_show_errors.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
CI

**Review by zhoxing-ms — APPROVED** [review]

**Review by yanzhudd — APPROVED** [review]

File: scripts/ci/automation_full_test.py [hunk 1]
```diff
@@ -441,13 +446,13 @@ def get_pipeline_result(test_result_fp, pipeline_result):
...
                     break
-                for i in pipeline_result[unique_job_name]['Details'][0]['Details'][0]['Details']:
+                for i in pipeline_result[unique_job_name]['Details'][0]['Details'][0]['Details'][0]['Details']:
                     if i['Module'] == module:
                         i['Status'] = 'Failed'
                         i['Content'] = build_markdown_content(state, test_case, message, line, i['Content'])
                         break
             else:
-                for i in pipeline_result[unique_job_name]['Details'][0]['Details'][0]['Details']:
+                for i in pipeline_result[unique_job_name]['Details'][0]['Details'][0]['Details'][0]['Details']:
```
Comment (jsntcy) [inline scripts/ci/automation_full_test.py:455]: What's the purpose to add `[0]['Details']`?

Comment (wangzelin007) [reply scripts/ci/automation_full_test.py:455]: image
This is due to the data structure we worked out with the azure-powershell team.

**Review by bebound — APPROVED** [review]

File: scripts/ci/automation_full_test.py (hunk 1 above)
Comment (jsntcy) [reply scripts/ci/automation_full_test.py:455]: So CLI only has one element for each `Details` property, while there will be more than one element for `Details` property in azure PS?

Comment (wangzelin007) [reply scripts/ci/automation_full_test.py:455]: Yes, there are multiple elements for Details property in azure PS.
Please note that we also have multiple elements in the innermost layer. 
The reason why the other layers only have a single element is that we use multiple instance tests. 
If these tests are run on the same machine, then our details will also have multiple elements.

**Review by jsntcy — APPROVED** [review]

**Pull request merged**